backend/README.md

```diff
@@ -0,0 +1,93 @@
+# Forgettable - Backend
+
+This is the backend of Forgettable repository.
+
+## How to set up backend
+1. Make sure your current directory is `Forgettable/backend`
+2. Create .env under repo's `backend` folder
+    - `MONGODB_URI` should be set to a running mongodb instance url (should start with `mongodb+src://`)
+    - `PORT` should be set to 3001 (*that is, if proxy on `forgettable-frontend` is set to `http://localhost:3001/`)   
+    - `NODE_ENV` should be set to `test` if you are running `npm run test`
```

Comment:
From https://jestjs.io/docs/environment-variables, it says that JEST auto sets the NODE_ENV to `'test'` if it is not already. So we might not need to add this to `.env`?

Reply:
ahhh I didn't know about this, thanks!